pages/form-builder/edit/[[...params]].tsx

```diff
@@ -1,14 +1,38 @@
-import React, { ReactElement } from "react";
+import React, { ReactElement, useEffect } from "react";
 import { useTranslation } from "next-i18next";
+import { useRouter } from "next/router";
 import { NextPageWithLayout } from "../../_app";
 import { PageProps } from "@lib/types";
 import { getServerSideProps } from "../index";
 import { EditNavigation, Template, PageTemplate } from "@components/form-builder/app";
 import { Edit } from "@components/form-builder/app/edit";
+import { useTemplateStore } from "@formbuilder/store";
 
 const Page: NextPageWithLayout<PageProps> = () => {
   const { t } = useTranslation("form-builder");
   const title = `${t("gcFormsEdit")} — ${t("gcForms")}`;
+  const router = useRouter();
+
+  const { id, isPublished } = useTemplateStore((s) => ({
+    id: s.id,
+    isPublished: s.isPublished,
+  }));
+
+  useEffect(() => {
```

Comment:
Logic added here in case we hit the `/edit` route direct and not already handled by the server